ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -41,13 +41,7 @@
                 let body = action.payload?.body;
                 if (body) {
                     if (body.redirectUrl) {
-                        let redirectURL = self._path + '.guideThankYouPage.html';  //default ThankYouPage Path
-                        let redirectElement = document.querySelector('[name=":redirect"]');
-                        // check to prevent tampering of redirectURL from client
-                        if(redirectElement && redirectElement.value === body.redirectUrl) {
-                            redirectURL = body.redirectUrl;
-                        }
-                        window.location.href = redirectURL;
+                        window.location.href = body.redirectUrl;
```

Comment:
@ravise5 Can we use `encodeURIComponent` to encode the URL before setting href in location ?

Reply:
@rismehta 
`encodeURIComponent` should be used for individual components of a URL like query parameters etc. 
It is not advisable to use on the entire url. Since we are not aware of the structure of adobe sign url coming from server  should we do this?